conf/logback.xml

```diff
@@ -21,6 +21,7 @@
     </appender>
 
     <logger name="connectors" level="DEBUG"/>
+    <logger name="controllers" level="DEBUG"/>
```

Comment:
Do we need to add anything for services here?

Reply:
service does not log anything interesting yet